extensions/resource-metrics/resource-metrics-extention/ui/src/Metrics/client.ts

```diff
@@ -46,3 +46,5 @@ export function getHeaders({
     "Argocd-Project-Name": `${project}`,
   };
 }
+
```

Comment:
please remove added spaces at the end of the file

Reply:
I removed the extra space from the end of the file.